app/javascript/packs/tiny_mce.js

```diff
@@ -0,0 +1,474 @@
+import tinyMCE from 'tinymce/tinymce';
+import 'tinymce/models/dom';
+import 'tinymce/icons/default';
+import 'tinymce/themes/silver';
+
+//import 'tinymce/plugins/custom_image_toolbar';
+import 'tinymce/plugins/table';
+import 'tinymce/plugins/autosave';
+import 'tinymce/plugins/autoresize';
+//import 'tinymce/plugins/customimageuploader';
+import 'tinymce/plugins/link';
+import 'tinymce/plugins/advlist';
+import 'tinymce/plugins/codesample';
+import 'tinymce/plugins/autolink';
+import 'tinymce/plugins/lists';
+import 'tinymce/plugins/charmap';
+//import 'tinymce/plugins/hr';
+import 'tinymce/plugins/anchor';
+import 'tinymce/plugins/searchreplace';
+import 'tinymce/plugins/wordcount';
+import 'tinymce/plugins/visualblocks';
+import 'tinymce/plugins/visualchars';
+import 'tinymce/plugins/insertdatetime';
+import 'tinymce/plugins/nonbreaking';
+import 'tinymce/plugins/save';
+import 'tinymce/plugins/directionality';
```

Comment:
I think for plugins we can make a config with a list of names. And just iterate through the list.